README.md

```diff
@@ -43,59 +58,81 @@ After your Lambda@Edge environment is prepared, install the starter kit:
-
 ### Initialization
 
-In the `src` folder of the starter kit, you'll find three JavaScript files:
+In the `src` folder of the starter kit, you will find two TypeScript files:
 
-- `src/viewer_request.js` contains sample code that fetches and caches the client configuration, initializes the Kameleoon SDK with this configuration, and modifies the outgoing request by adding the User ID cookie and feature flag value to the header.
-- `src/viewer_response.js` file contains sample code that fetches the User ID cookie from the incoming request and attaches the cookie to the outgoing response.
-- `src/helpers.js` contains some additional platform-specific code that demonstrates common features of the Kameleoon SDK.
+- `src/handler.ts` contains sample code that initializes and caches the Kameleoon SDK and retrieves the feature flag variation.
+- `src/visitorCodeManager.ts` file contains a custom implementation of Kameleoon Visitor Code Manager, which provides an ability to work with Kameleoon `visitorCode` at the edge.
 
 You'll use these files to initialize your environment:
 
-1. Navigate to `src/viewer_request.js` and update the `YOUR_SITE_CODE`, `YOUR_FEATURE_KEY`, `YOUR_CLIENT_ID` and `YOUR_CLIENT_SECRET` values with your [Kameleoon credentials](https://help.kameleoon.com/api-credentials).
-1. Review or adjust the feature flags or experiments you've configured with the Kameleoon NodeJS SDK, and hook into the lifecycle events by inserting your desired logic into the switch-case statements in `src/viewer_request.js` and `src/viewer_response.js`. For example, you can change headers, cookies, and more.
-1. Run `npm run build:viewer_request` to use Rollup to bundle the source code of `src/viewer_request.js` into a `dist/dist.zip` file that you'll import into Lambda. The `dist.zip` file should be roughly **~49kb** in size for `src/viewer_request.js` assuming you have not made any additional changes.
-1. Upload the request `dist/dist.zip` file into Lambda using one of these options:
-   - **GUI**: Go to your AWS Lambda console, select the function associated with your Lambda environment, and import the `dist.zip` file. After you upload it, there should now be a minified `index.js` file located inside of your Lambda function's **Code Source** section.
+1. Navigate to `src/handler.ts` and update the `SITE_CODE`, `CLIENT_ID`, `CLIENT_SECRET` and `FEATURE_KEY` with your [Kameleoon credentials](https://help.kameleoon.com/api-credentials) and feature flag data collected on Kameleoon Platform.
+1. Review or adjust the feature flags or experiments you've configured with the Kameleoon NodeJS SDK, and hook into the lifecycle events by inserting your desired logic in `src/handler.ts`. For example, you can change headers, cookies, and more.
+1. Run `npm run build:lambda` to bundle the source code into a `dist/handler.zip` file that you'll import into Lambda.
+1. Upload the handler `dist/handler.zip` file into Lambda using one of these options:
```

Comment:
I would leave "handler" out here. It's redundant since it's already in the file name and it makes the sentence more complicated to parse.